content/en/continuous_integration/setup_pipelines/jenkins.md

```diff
@@ -10,6 +10,11 @@ further_reading:
       text: "Troubleshooting CI"
 ---
 
+## Compatibility
+
+Supported Jenkins versions:
+* Jenkins >= 2.164.1
+
```

Comment:
Probably, we want to update Jenkins instructions with this PR (https://github.com/DataDog/documentation/pull/11007)

Reply:
Yes, once that's merged I'll rebase this PR